ci-operator/step-registry/upi/install/gcp/by-infra-manager/upi-install-gcp-by-infra-manager-commands.sh

```
@@ -0,0 +1,729 @@
+
+# check if OCP version will be equal to or greater than the minimum version
+# $1 - the minimum version to be compared with
+# return 0 if OCP version >= the minimum version, otherwise 1
+function version_check() {
+  local -r minimum_version="$1"
+  local ret
+
+  dir=$(mktemp -d)
+  pushd "${dir}"
+
+  cp ${CLUSTER_PROFILE_DIR}/pull-secret pull-secret
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)